118.md

```diff
@@ -0,0 +1,89 @@
+# NIP-118
+
+## Polls
+
+`draft` `optional`
+
+This NIP defines the event scheme that describe Polls on nostr.
+
+The NIP makes the following tradeoffs.
+
+1. All votes are public.
+2. All responses are editable (latest by timestamp considered last response)
+
+## Events
+
+### Poll Content
+
+The poll event is defined as a `kind:1068` event. Major tags in the poll event are:
+
+- **label**: The accompanying text of the poll.
+- **option**: The option tags contain an OptionId(any alphanumeric) field, followed by an option label field.
+- **relays**: Set of relays that the poll is expecting respondents to respond on.
+- **settings**: A JSON stringified object that may hold poll level configs and client specific configs.
```

Comment:
Why the JSON? I haven't seen it as the value of the tag.

Reply:
carried it over from forms where i use it for Ui settings like imageurl etc. Thought I might reuse the same config for the pollType field, but I think the PollType field in the tags also makes sense.

Reply:
I would get rid of it. Add all the extra settings as tags.